packages/web/components/alert/prettify.ts

```
@@ -27,6 +27,34 @@ const regexRejectedTx = /Request rejected/;
+  if (!message) return;
+
+  const match = message.match(regexOverspendError);
+  if (!match) return;
+
+  const [, wouldSpendTotal, limit] = match;
+
+  if (!wouldSpendTotal || !limit) return;
+
+  try {
+    // Validate that extracted values are valid numbers
+    if (isNaN(Number(wouldSpendTotal)) || isNaN(Number(limit))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)